lldb/source/Plugins/Protocol/MCP/ProtocolServerMCP.cpp

```diff
@@ -244,6 +253,7 @@ ProtocolServerMCP::HandleData(llvm::StringRef data) {
 protocol::Capabilities ProtocolServerMCP::GetCapabilities() {
   protocol::Capabilities capabilities;
   capabilities.tools.listChanged = true;
+  capabilities.resources.listChanged = false;
```

Comment:
Should we support sending notifications when a debugger/target are added/removed? Otherwise the client could get out of sync with the state of lldb.

Maybe as a FIXME/TODO if we can't easily do that now.